src/android/CHIPTool/app/src/main/res/layout/barcode_fragment.xml

```diff
@@ -4,12 +4,45 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <com.google.chip.chiptool.setuppayloadscanner.CameraSourceView
-        android:id="@+id/camera_view"
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight="9">
```

Comment:
layout_weight is a bit hacky. I modified this xml to use ConstraintLayout, can you patch this in instead?

https://gist.github.com/austinh0/ef02951223826d785b255d6799d4cef5

Reply:
OK, I will merge that, thanks.

Reply:
updated at b575d5b18bde7e2c0ef34d531f034e6432c6de7f